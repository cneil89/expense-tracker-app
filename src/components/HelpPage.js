import React from 'react'

const HelpPage = () => (
  <div>
    LOOK HERE FOR HELP!
  </div>
);

export default HelpPage