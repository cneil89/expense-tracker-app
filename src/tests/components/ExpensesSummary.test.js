import React from 'react'
import { shallow } from 'enzyme'
import { ExpensesSummary } from '../../components/ExpensesSummary'

test('Should render component with 1 expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />)
  expect(wrapper).toMatchSnapshot()
})

test('Should render component with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={876543342} />)
  expect(wrapper).toMatchSnapshot()
})