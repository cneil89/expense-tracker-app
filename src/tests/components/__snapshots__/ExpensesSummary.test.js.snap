// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render component with 1 expenses 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`Should render component with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $8,765,433.42
  </h1>
</div>
`;
